config/trees.yaml

```diff
@@ -188,144 +188,188 @@ build_variants:
 
 build_configs:
 
+  mainline: &mainline
+    tree: mainline
+    branch: 'master'
+
+  mainline-standard: &mainline-standard
```

Comment:
I think we are missing an option for *all* supported architectures that should be used in:
* mainline
* next
* stable and stable-rc
* sashal/linus-next

Reply:
@padovan, if we do not add the architectures property filter, a given tree would run jobs on every architecture we support. This is the default behaviour.

Reply:
Done.

Reply:
I wonder if the "net-next" tree should also run on every architecture. Same for "netdev-testing" tree. But, perhaps only the former would benefit from running on every platform we cover.

Reply:
I will move "net-next" to run on every architecture. Seems only prudent.

Reply:
Keep netdev-testing out of it for now. We are running some specific tests on it for the time being.

Reply:
Done.